fraud_online/1_feature_groups.ipynb

```diff
@@ -5,12 +5,14 @@
    "id": "dbca0c8a",
    "metadata": {},
    "source": [
-    "# <span style=\"font-width:bold; font-size: 3rem; color:#1EB182;\"><img src=\"../images/icon102.png\" width=\"38px\"></img> **Hopsworks Feature Store** </span><span style=\"font-width:bold; font-size: 3rem; color:#333;\">- Part 01: Load, Engineer & Connect</span>\n",
```

Comment:
You don't have to remove the image entirely, see the batch notebooks, we have the images embedded there. Can we do the same here?